index.html

```diff
@@ -0,0 +1,156 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css">
+</head>
+<body>
+    <header class = "top">
+        <div class="head">
+           <img src="Logo.svg" class="logo">
+            <img src="Union.svg" class="union">
```

Comment:
- [ ] KIndly wrap your hamburger icon into the `nav` tag because of semantics reasons, and the right position for convention and user experience for mobile devices. :wink:

Reply:
@levy002 resolved